packages/runtime/src/internal/CustomTemplates/replaceSlotWithSlottedBricks.ts

```
@@ -0,0 +1,75 @@
+import type {
+  BrickConf,
+  SlotsConfInTemplate,
+  UseSingleBrickConf,
+} from "@next-core/types";
+import { hasOwnProperty } from "@next-core/utils/general";
```

Biome

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)